src/models/TraktItem.ts

```diff
@@ -61,4 +70,22 @@ export class TraktItem implements ITraktItem {
 			releaseDate: item.releaseDate,
 		};
 	};
+
+	static save = (item: TraktItem): SavedTraktItem => {
+		return {
+			...TraktItem.getBase(item),
+			syncId: item.syncId,
+			watchedAt: item.watchedAt?.unix(),
+			progress: item.progress,
+		};
+	};
+
+	static load = (savedItem: SavedTraktItem): TraktItem => {
+		const options: ITraktItem = {
+			...savedItem,
+			watchedAt:
+				typeof savedItem.watchedAt !== 'undefined' ? moment(savedItem.watchedAt * 1e3) : undefined,
```

Comment:
What does ie3 mean?

Reply:
10^3 = 1000, basically just converting from seconds to milliseconds.

Reply:
Right. Strange notation. In my mind it would be 10e3, but I see that you are right.